python/tvm/tir/schedule/schedule.py

```diff
@@ -2111,6 +2112,71 @@ def after_unannotate(a: T.handle, b: T.handle) -> None:
             self, block_or_loop, ann_key
         )
 
+    ########## Schedule: Layout transformation ##########
+
+    def transform_layout(
```

Comment:
Shall we add @type_checked to this API to be consistent?

Reply:
+1